vdev/src/commands/cli.rs

```diff
@@ -0,0 +1,41 @@
+use anyhow::Result;
+use clap::{Parser, Subcommand};
+use clap_verbosity_flag::{InfoLevel, Verbosity};
+
+/// Vector's unified dev tool
+#[derive(Parser, Debug)]
+#[command(version, bin_name = "vdev", disable_help_subcommand = true)]
+pub struct Cli {
+    #[clap(flatten)]
+    pub verbose: Verbosity<InfoLevel>,
+
+    #[command(subcommand)]
+    command: Commands,
+}
+
+#[derive(Subcommand, Debug)]
+enum Commands {
+    Build(super::build::Cli),
+    Complete(super::complete::Cli),
+    Config(super::config::cli::Cli),
+    Exec(super::exec::Cli),
+    Int(super::int::cli::Cli),
```

Comment:
Is there any way (in `clap`) to keep commands like this as the full name but allow abbreviations? For example, the `btrfs` command has a `filesystem` subcommand, but any unique substring like `fi` is allowed.

Reply:
https://docs.rs/clap/latest/clap/builder/struct.Command.html#method.infer_subcommands I believe.

Reply:
Done!